src/proto/FIO.proto

```diff
@@ -0,0 +1,88 @@
+syntax = "proto3";
+
+package TW.FIO.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// A public blockchain address, such as {"BTC", "bc1qvy4074rggkdr2pzw5vpnn62eg0smzlxwp70d7v"}
+message PublicAddress {
+    string token_code = 1;
+    string address = 2;
+}
+
+// Different Actions
+message Action {
+    // Action for registering a FIO name; register_fio_address
+    message RegFioAddress {
+        // The FIO name to be registered. Ex.: "alice:trust"
+        string fio_address = 1;
```

Comment:
just `address` ?

Reply:
There are 2 kinds of addresses in this msg, fio and other chain; prefix is good to prevent mixup; keeping.